dependencies-path.yaml

```diff
@@ -0,0 +1,54 @@
+- dependency: "etcd"
+  refPaths:
+  - build/root/WORKSPACE
+  - cluster/gce/config-test.sh
+  - cluster/gce/manifests/etcd-empty-dir-cleanup.yaml
+  - cluster/gce/manifests/etcd.manifest
+  - cluster/gce/upgrade-aliases.sh
+  - cluster/images/etcd-empty-dir-cleanup/Makefile
+  - cluster/images/etcd/Makefile
+  - cluster/images/etcd/README.md
+  - cluster/images/etcd/migrate-if-needed.sh
+  - cmd/kubeadm/app/constants/constants.go
+  - cmd/kubeadm/app/constants/constants_test.go
+  - hack/lib/etcd.sh
+  - staging/src/k8s.io/sample-apiserver/artifacts/example/rc.yaml
+  - test/e2e/apimachinery/aggregator.go
+  - test/e2e/framework/nodes_util.go
+  - test/kubemark/start-kubemark.sh
+
+- dependency: "docker"
```

Comment:
we also have the concept of a minimal docker version somewhere in the kubelet(?)
i forgot where exactly.

Reply:
Yes, these are min. versions of docker we tests kubernetes against, but for now didn't want to include it

Reply:
Yes, these are min. versions of docker we tests kubernetes against, but for now this isn't in the scope of this PR

Reply:
do we have a version number for `docker` in all these files?